src/components/SUE/SUEStatus.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { ViewStyle } from 'react-native';
+
+import { RegularText } from '../Text';
+import { Wrapper } from '../Wrapper';
+import { normalize } from '../../config';
+
+enum StatusTypes {
+  InBearbeitung = 'In Bearbeitung',
+  Offen = 'Offen',
+  Unbearbeitet = 'Unbearbeitet'
+}
+
+const defaultBackgroundColors = (status: string) => {
+  switch (status) {
+    case StatusTypes.InBearbeitung:
+      return '#FECDCA';
+    case StatusTypes.Offen:
+      return '#FEF0C7';
+    case StatusTypes.Unbearbeitet:
+      return '#D1FADF';
+    default:
+      return 'rgb(0, 0, 0)';
+  }
+};
+
+const defaultTextColors = (status: string) => {
+  switch (status) {
+    case StatusTypes.InBearbeitung:
+      return '#B42318';
+    case StatusTypes.Offen:
+      return '#B54708';
+    case StatusTypes.Unbearbeitet:
+      return '#027A48';
+    default:
+      return 'rgb(0, 0, 0)';
+  }
+};
+
+export const SUEStatus = ({
+  backgroundColors,
+  containerStyle,
+  status,
+  textColors
+}: {
+  backgroundColors: string;
+  containerStyle: ViewStyle;
```

Comment:
passing a `containerStyle` is required?

Reply:
I think it is necessary for this to be adjustable via the server in the future.

Reply:
yes, as we talked about other stuff related to the styles, i agree